app/graphql/loaders/association_loader.rb

```diff
@@ -6,6 +6,7 @@ def self.validate(model, association_name)
     end
 
     def initialize(model, association_name)
+      super()
```

Comment:
We usually omit braces on method calls without arguments
```suggestion
      super
```